### PR TITLE
Adds google-cloud python lib

### PR DESCRIPTION
**Pull request by arvy**

### CODE REVIEW

**Comment by arvy** [comment]
..merged by accident

**Comment by arvy** [comment]
@yazgoo Have a look at this and approve "a posteriori"

File: Dockerfile [hunk 1]
```diff
@@ -56,6 +56,7 @@ RUN set -ex \
     && pip install ndg-httpsclient \
     && pip install pyasn1 \
     && pip install google-api-python-client==1.5.3 \
+    && pip install google-cloud \
```
Comment (yazgoo) [inline Dockerfile]: Wouldn't we like to pin a gcloud version here.

Comment (arvy) [inline Dockerfile]: good idea - i'll take the latest one

Comment (arvy) [inline Dockerfile]: Undid this merge - here's the new PR
https://github.com/ebuzzing/tool-airflow/pull/6

**Pull request merged**